### PR TITLE
style sermon meta

### PR DESCRIPTION
**Pull request by narthur**

### CODE REVIEW

**Comment by narthur** [comment]
Perhaps related to sass module issue:

- https://github.com/vercel/next.js/issues/11474
- https://github.com/vercel/next.js/issues/10468
- https://github.com/vercel/next.js/pull/11795

File: pages/[language]/sermons/[id].js [hunk 1]
```diff
@@ -19,36 +20,38 @@ export default function Post({ sermon }) {
...
-                                </ul>
-                            </div>
+                <div>
+                    <div className={styles.meta}>
+                        {imageSrc ? <img src={imageSrc} alt={imageAlt}/> : null}
+                        <div>
+                            <h1>{sermon.title}</h1>
+                            <ul className={styles.speakers}>
+                                {sermon.persons.map(speaker => {
+                                    return <li>{speaker.name}</li>
```
Comment (mattleff) [inline pages/[language]/sermons/[id].js:30]: Looks like we need a `key` prop.

File: pages/[language]/sermons/[id].js [hunk 2]
```diff
@@ -19,36 +20,38 @@ export default function Post({ sermon }) {
...
                         </div>
+                    </div>
 
-                        {sermon.recordingDate ? <p>{(new Date(sermon.recordingDate)).toLocaleDateString()}</p> : null}
-                        {sermon.audioFiles.map(file => {
-                            return <div><audio controls src={file.url} preload={'metadata'}>Your browser doesn't support this player.</audio></div>
-                        })}
+                    {sermon.recordingDate ? <p>{(new Date(sermon.recordingDate)).toLocaleDateString()}</p> : null}
+                    {sermon.audioFiles.map(file => {
+                        return <div>
```
Comment (mattleff) [inline pages/[language]/sermons/[id].js:38]: Looks like we need a `key` prop.

**Review by mattleff — APPROVED** [review]
Looks good to me!

**Pull request merged**